lib/tests-from-pull-request.js

```diff
@@ -1,32 +1,30 @@
-const fs = require('fs')
-const readline = require('readline')
+module.exports = getTestsFromPR;
 
-module.exports = getTestsFromPR
+function getTestsFromPR(context) {
+  const pull = context.issue();
+  const {pullRequests} = context.github;
 
-function getTestsFromPR (context) {
   return Promise.resolve()
-    .then(() => ['Hi Or'])
-  // .then(() => _getTestsFromPR(context))
-}
-
-function _getTestsFromPR (context) {
-  const pull = context.issue()
-  const {repos, pullRequests} = context.github
-  // todo: code to get file contect
-  let newTestLines = []
-  let PRfiles
-  return pullRequests.getFiles(pull)
+    .then(pullRequests.getFiles(pull))
     .then((res) => res.data)
-    .then((files) => PRfiles = files)
-    .then(() => console.log('FILES FROM PR:\n', PRfiles))
+    .then((files) => files.map((file) => file.patch))
+    .then((patches) => patches.join('\n'))
+    .then(_prPatch);
+}
 
-  // let line = ''
-  // if (line.startsWith('+')) {
-  //   line = line.slice(1).trim() // remove the + and spaces
-  //   if (line.trim().startsWith('it(')) {
-  //     newTestLines.push(line)
-  //   }
-  // }
+function _prPatch(prPatch) {
```

Comment:
This will give the entire information about this function. both the fact that it gets you the **test names**, and also the fact that it needs to get the **prPatch** as input